tokamak/2-currents/BOUT.inp

```diff
@@ -0,0 +1,273 @@
+
+first = FFT
+second = FFT
+upwind = W3
+
+###################################################
+# Time-integration solver
+
+[solver]
+# Note: If evolving neutrals, need preconditioning
+type = cvode
+use_precon = true
+
+maxl = 10
+
+ATOL = 1.0e-10  # absolute tolerance
+RTOL = 1.0e-5   # relative tolerance
+mxstep = 1000000  # Maximum internal steps per output
+
+##################################################
+# Electrostatic potential solver
+
+[phiSolver]
+inner_boundary_flags = 0   # Zero-gradient inner boundary
+outer_boundary_flags = 16  # INVERT_SET, setting outer boundary
+
+[laplacexy]  # 2D solver in X-Y
+pctype = hypre    # Preconditioner
+
+atol = 1e-12
+rtol = 1e-8
+
+# Note: These boundary flags should be consistent with phiSolver
+core_bndry_dirichlet = false
+pf_bndry_dirichlet = true
+y_bndry_dirichlet = false
+
+include_y_derivs = true  # Include the y components in the inversion
+
+[aparSolver]
+inner_boundary_flags = 0
+outer_boundary_flags = 0
+
+all_terms = true
+nonuniform=true
+
+###################################################
+# Model settings
+
+[Hermes]
+
+#######################
+# Output variables
+output_ddt = false # Output time derivatives
+verbose = true     # Output more derived quantities
+
+#######################
+# Numerical dissipation
+
+vepsi_dissipation = true  # Parallel dissipation on Ve-Vi
+vort_dissipation = false  # Parallel dissipation of vorticity oscillations
+phi_dissipation = true    # Parallel dissipation of potential oscillations
+
+radial_buffers = true   # Turn on dissipation close to boundaries
+radial_inner_width = 4  # Number of cells on inner boundary
+radial_outer_width = -1  # Number of cell on outer boundary. < 0 -> no buffer
+radial_buffer_D = 1     # Damping coefficient in boundary
+
+#######################
+# Flux limiters
+kappa_limit_alpha = 0.2 # SOLPS style heat flux limiter
+eta_limit_alpha   = 0.5 # SOLPS style viscosity limiter
+
+#######################
+# Electric field and Ohm's law
+electromagnetic = true  # Electromagnetic? Otherwise electrostatic
+FiniteElMass    = true  # Finite electron mass?
+
+# Electrostatic potential
+split_n0 = false  # Solve n=0 electrostatic potential separately?
+split_n0_psi = false  # Solve n=0 Apar component separately?
+
+phi_boundary_relax = true  # Relax the boundary towards zero gradient
+phi_boundary_timescale = 1e-5  # Timescale for relaxation [seconds]
+
+#######################
+# Terms included in the model
+
+# NOTE: all currents switched off for fluid run
+j_diamag = true
+j_par = true
+
+pe_par         = true # Parallel pressure gradient: Pe <-> Psi
+resistivity    = true  # Resistivity: Psi -> Pe
+thermal_flux   = true
+thermal_force  = true
+electron_viscosity = true
+ion_viscosity = false     # Viscous terms in vorticity equation
+ion_viscosity_par = true  # Parallel diffusion of ion momentum
+thermal_conduction = true   # Parallel heat conduction
+poloidal_flows = false   # Include ExB flows in the X-Y plane?
```

Comment:
If we do `split_n0 = true`, I guess this should be `true` as well?

Reply:
Not sure if they need to be related. Perhaps.